composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.6",
-        "http-interop/http-middleware": "^0.5"
+        "http-interop/http-server-middleware": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^4.8|^5.0",
```

Comment:
PHP 4.x can no longer be used with PHP 7. This should be changed to `^5.7|^6.0`.